docs/source/user_guide/user_guide/static.md

```diff
@@ -0,0 +1,5 @@
+# ti.static
+
+ti.static will evaluate expressions and function results at compilation time, rather than at kernel execution time.
```

Comment:
I can also be used to create aliases, which I think is useful and therefore "important" to mention.